content/en/examples/application/job/job-sidecar.yaml

```diff
@@ -12,10 +12,8 @@ spec:
           volumeMounts:
             - name: data
               mountPath: /opt
-      initContainers:
-        - name: logshipper
+        - name: sidecar-container
```

Comment:
Same comment. Do not modify the existing examples. Instead, create a new section for the legacy way of defining sidecars.

Reply:
Done.